Go Workshop API.postman_collection.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Thanks for adding this file.

Reply:
Thanks for the suggestion of adding a file like that 😆 I added it with postman because it's the tool I use, but I'll give it a try to the tool you suggested